README.md

```diff
@@ -46,7 +46,7 @@ Probes processor, sysfs, and KVM for AMD SEV, SEV-ES, and SEV-SNP related featur
 
 ```console
 $ sevctl ok {sev, es, snp}   // Probes support for the generation specified.
-$ sevctl ok                  // Probes for SEV, SEV-ES, and SEV-SNP support.
```

Comment:
I kind of feel like the original comment was easier to understand. Perhaps adding an `(autodetected)` to the end of it would be sufficient?

Reply:
I read that as `Probes for SEV, SEV-ES, and SEV-SNP support regardless of the generation you're running on`, which would no longer be the case with this update.

What about `Probes for SEV, SEV-ES, and SEV-SNP based on the autodetected generation the host CPU is running on`?